shared/lib/deep-links/routes/onboarding.ts

```diff
@@ -0,0 +1,11 @@
+import { Route, ZENDESK_URLS } from './route';
+
+export default new Route({
+  pathname: '/onboarding',
+  getTitle: (_: URLSearchParams) => 'deepLink_theOnboardingPage',
+  handler: function handler(_params: URLSearchParams) {
+    return {
+      redirectTo: new URL(ZENDESK_URLS.IMPORTED_ACCOUNTS_PRIVATE_KEY),
```

Comment:
Interestingly enough this constant has a broken value. See this ticket: https://github.com/MetaMask/metamask-extension/issues/38494

Can we fix in this PR?